### ISSUE TITLE
ability to move an issue from one repo to another.

### ISSUE THREAD

**Issue by iamhabitat** [waffleio/waffle.io#1729]

**Comment by thompaul** [comment]
+1

**Comment by mariushm** [comment]
:+1:

**Comment by homeyer** [comment]
We won't be adding this directly in Waffle anytime soon. I'd recommend checking this tool out in the meantime: https://github-issue-mover.appspot.com/

Are you usually moving issues because you have multiple repos on one Waffle board, and accidentally created the issue in the wrong repo?

**Comment by thompaul** [comment]
@homeyer Yes, this is the reason we want/need this feature. In some cases it's not clear to the issue creator which repo the issue belongs to - this is especially true for non-devs. We'd love to be able to reassign the issue to the right repo when the dev path becomes clearer. A corollary to this would be to have a default parking lot space to have issues that aren't tied to any repo - though this would take away the luxury of having github be the gatekeeper of all issues.

**Comment by teu** [comment]
+1 needed like water in the Sahara

**Comment by iamhabitat** [comment]
+1

**Comment by ashumz** [comment]
[+1](https://app.intercom.io/a/apps/fb774a034de340cde33822ae0c071ad60efe899f/inbox/ashley@waffle.io/conversations/846814840)

**Comment by ashumz** [comment]
Hi all, 

Since we're not planning to do this work in the near future, I figured I'd add a couple of ways our users are dealing with this issue today: 

1. For multirepo boards, you can rename sources in Settings->Sources to be more user friendly than that of the repo path. For non-devs, this might be helpful for helping to make sure the issue is dropped in the right place!

2. You can add a repository for issues only that have not yet been triaged -- you might even name it 'parking lot' @thompaul :) - Issues would need to be a bit more defined before being added to the proper repo in these cases, anyways. There would be some amount of duplication, but perhaps you could have a label in the parking lot repo for 'Triage' that can be sorted by on a regular cadence, and issues can be moved appropriately at that time. 

Just a few suggestions for working around this in the meantime! Happy to consider why these options would or wouldn't work for those still looking for this functionality in the future.

Best, 

Ashley

**Comment by freeall** [comment]
I've worked in several places where this have been a very useful feature, but that's still lacking - both from github and tools like waffle.

In my experience this pattern is almost always used in larger projects:

- Several repos per project, like web/api/picture-scaler/parser, etc. etc.
- Very often issues would start in "web". Imagine something like "Upload button doesn't work". But after debugging you notice that it's actually an error in "api", who then finds out it was a problem with the "picture-scaler".

In these cases it would be incredibly useful to - somehow - move the issue to another repo. Moving could mean to simply include a reference to the old issue or it could be to copy the comments, etc, etc over if that's possible.

I hope you'll think about adding this. It could be very, very useful and I know many that would love it.

**Comment by fatima-alansari** [comment]
Please add this :(

**Comment by kleberpinel** [comment]
+1

**Comment by txtm** [comment]
+1

**Comment by thehufbro** [comment]
Soooo, it's been a few years, I am wondering if this issue has received any further consideration.

**Comment by amitpagrawal** [comment]
We are also interested in this feature, it's really bottleneck, Please consider this, can't rely on other third party tools.
